libshortfin/CMakeLists.txt

```diff
@@ -127,6 +123,9 @@ if (NOT SHORTFIN_IREE_SOURCE_DIR AND SHORTFIN_BUNDLE_DEPS)
 endif()
 
 if(SHORTFIN_IREE_SOURCE_DIR)
+  if(SHORTFIN_ENABLE_ASAN)
```

Comment:
Necessary? I would have expected that the -fsanitize=address from the parent project applied to the sub project.

Reply:
It might be necessary if building the compiler, but it is not for the runtime. Removed it for now.